generators/server/templates/gradle/_profile_dev.gradle

```diff
@@ -31,7 +31,7 @@ task gulpConstantDev(type: GulpTask) {
     args = ["ngconstant:dev", "--no-notification"]
 }
     <%_ } else { _%>
-task webpack(type: <%= clientPackageManager.charAt(0).toUpperCase() + clientPackageManager.slice(1) %>Task, dependsOn: '<%= clientPackageManager %>_install') {
+task webpackBuildDev(type: <%= clientPackageManager.charAt(0).toUpperCase() + clientPackageManager.slice(1) %>Task) {
```

Comment:
In maven the webpack profile installs local node and yarn|npm so instead of depending on npmInstall|yarnInstall we should see if we can depend on a task to make sure local versions are installed if possible otherwise I'm fine with it